docs/slides/gcp/hcp-terraform/index.html

```diff
@@ -7,7 +7,7 @@
 <html>
 
 <head>
-  <title>Introduction to Terraform OSS on AWS</title>
+  <title>HCP Terraform on Google Cloud Platform</title>
```

Comment:
the abbreviation is used in other places, should change to GCP

Reply:
Done.